synthetics_plugin-health-scoring.tf

```diff
@@ -22,3 +22,28 @@ resource "datadog_synthetics_test" "plugin_health_scoring" {
 
   status = "live"
 }
+
+resource "datadog_synthetics_test" "plugin_health_scoring" {
```

Comment:
Either rename the resource
```suggestion
resource "datadog_synthetics_test" "plugin_health_scoring_probes" {
```

Or either remove the existing one and keep only this new one.

Reply:
Good point! I was not really awaken. Thanks, I'm adding your suggestion